asv/graph.py

```diff
@@ -303,21 +301,22 @@ def resample_data(val):
     if len(val) < RESAMPLED_POINTS:
         return val
 
-    min_time = min(x[0] for x in val)
-    max_time = max(x[0] for x in val)
-    step_size = int((max_time - min_time) / RESAMPLED_POINTS)
+    min_revision = min(x[0] for x in val)
+    max_revision = max(x[0] for x in val)
+    step_size = int((max_revision - min_revision) / RESAMPLED_POINTS)
 
     if step_size == 0:
-        step_size = max_time - min_time + 1
+        step_size = max_revision - min_revision + 1
 
     # This loop cannot use xrange, because xrange on Python2 on
     # 32-bit systems can only deal with 32-bit integers, and
     # Javascript timestamps (1000*unix_timestamp) handled here
     # overflow this range
+    # FIXME: Use xrange
```

Comment:
`for i in xrange(min_revision, max_revision, step_size)`  (iow just revert cc6accaee3, d88095 since it's not needed any more)